graviti/client/branch.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+
+"""Interfaces about the branch."""
+
+from typing import Optional
+
+from requests.models import Response
+
+
+def create_branch(  # pylint: disable=unused-argument
+    url: str,
+    access_key: str,
+    dataset_id: str,
+    name: str,
+    commit: str,
+) -> Response:
+    """Get the response of OpenAPI `POST /v1/datasets{id}/branches`.
+
+    Arguments:
+        url: The URL of the graviti gas website.
+        access_key: User's access key.
+        dataset_id: Dataset ID.
+        name: The name of the branch.
+        commit: The information to locate the specific commit, which can be the commit id,
+            the branch name, or the tag name.
+
+    Return:
```

Comment:
do we need `Example` section here?

Reply:
Yes, should we add the examples in the design phase or in the implementation phase?

Reply:
I think it is OK to put it to implementation phase